.docker-mate/systems/wordpress/.env

```diff
@@ -1,3 +1,3 @@
 export URL_BACKEND=wp-admin
 
-export PHP_VERSION=8.0
+export PHP_VERSION=8.1
```

Comment:
A lot of plugins are still depending on PHP 8.0 and will throw errors with 8.1

Reply:
> A lot of plugins are still depending on PHP 8.0 and will throw errors with 8.1. You think this is right move?

Active support for PHP 8.0 ends 26 Nov 2022: https://www.php.net/supported-versions.php
So yes, I think we should move ahead with this change as it's easy enough for users to use a different PHP version than waht we provide as default.